lib/moon/design/accordion/content.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Moon.Design.Accordion.Content do
   @moduledoc false
 
-  use Surface.Component, slot: "content"
+  use Moon.StatelessComponent, slot: "content"
```

Comment:
Question about  Moon.StatelessComponent - should I also be using this instead of Surface.Component (in e.g. Modal.Panel, Modal.Header etc)?

Reply:
yes for now. it also imports some common functions, e.g. merge